### DISCUSSION TITLE
miCS error

### DISCUSSION THREAD

**Discussion by yunll** [deepspeedai/DeepSpeed#3460] [Q&A]
I use miCS config like this
![image](https://user-images.githubusercontent.com/15276878/236410859-c60596ab-d5f0-48d0-97e4-1181fb0bebe9.png)

I get this error
![image](https://user-images.githubusercontent.com/15276878/236411361-48205e5e-c914-4242-9628-6cf71e893245.png)

deepspeed version
0.9.2

**Comment by learning-chip** [comment]
See https://github.com/microsoft/DeepSpeed/discussions/3438#discussioncomment-5842921
